koji/README.md

```diff
@@ -0,0 +1,20 @@
+---
+title: koji
+homepage: https://github.com/its-danny/koji
+tagline: |
+  🦊 An interactive CLI for creating conventional commits. 
+---
+
+To update or switch versions, run `webi koji@stable` (or `@v2`, `@beta`, etc).
+
+## How to run `koji`
+
+```bash
+koji
+
+# with emoji
+koji --emoji
+
+# as a git hook (prepare-commit-msg)
+koji --hook
+```
```

Comment:
Again, I just need a little more info. Most people who come to webi are very much _NOT_ expert developers and probably wouldn't understand.

I'm guessing this runs whenever you run `git commit` with any options and rejects messages that don't include commits? Or asks you to add a prefix?

Just needs more detail. :)